readme: supported formats change (#349)
ouch-org/ouch: README.md

| Format    | `.tar` | `.zip` | `.gz` | `.xz`, `.lzma` | `.bz`, `.bz2` | `.lz4` | `.sz` | `.zst` |
|:---------:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|

> | Supported | ✓ | ✓¹ | ✓² | ✓ | ✓ | ✓ | ✓ | ✓² | ✓ |

Reviewer: Looks like we've been having one extra `✓` the whole time

```suggestion
| Supported | ✓ | ✓¹ | ✓² | ✓ | ✓ | ✓ | ✓² | ✓ |
```

✓: Supports compression and decompression.

> ✓¹: Due to `.zip` limitations, it doesn't support streaming formatting.

Reviewer: ```suggestion
✓¹: Due to limitations of `.zip`, it doesn't support streaming (de)compression.
```

> ✓²: Supported, and run in parallel.

Reviewer: maybe that's better? idk
```suggestion
✓²: Supported, and runs in parallel.
```

> ✓: Supports compression and decompression.
> ✓¹: Due to limitations of `.zip`, it doesn't support streaming (de)compression.
> ✓²: Supported, and compression runs in parallel.

Reviewer: ```suggestion
✓: Supports compression and decompression.

✓¹: Due to limitations of `.zip`, it doesn't support streaming (de)compression.

✓²: Supported, and compression runs in parallel.
```

Author: Oh yeah, Markdown, there's this weird thing where you can force a line break by adding `'\'` at the end of a line, `<br>` works too, because HTML, but if you skip 2 lines, it does add more padding.

Reviewer: skipping an extra line looks better than `\` and `<br>` to me in plain text, but if you don't like the extra padding, feel free to switch to one of the alternatives instead
